demo/iframe_resizer/run.py

```diff
@@ -11,21 +11,35 @@ def greet():
 
 
 with gr.Blocks() as demo:
-    with gr.Row(height=1500):
-        gr.Markdown("Scroll down to see UI.")
-    greet_btn = gr.Button("Trigger toast")
-    greet_btn.click(fn=greet)
+    with gr.Tab("Accordions"):
```

Comment:
Nice @pngwn ! Let's add this demo to the [list](https://github.com/gradio-app/gradio/blob/03024f8cae5d1a35be8fe6a988ae3917eefba662/scripts/copy_demos.py#L15) of deployed spaces so we can test for regressions before we release.

In order for the demo to work you should change the value of all the images to be the full path to the cheetah file

Reply:
i think i did it

Reply:
Yep looks good to merge!